factory/processComponents.go

```diff
@@ -1236,13 +1236,14 @@ func createNetworkShardingCollector(
 		return nil, err
 	}
 
-	psm, err := networksharding.NewPeerShardMapper(
-		cachePkPid,
-		cachePkShardID,
-		cachePidShardID,
-		nodesCoordinator,
-		epochStart,
-	)
+	arg := networksharding.ArgPeerShardMapper{
+		PeerIdPk:         cachePkPid,
```

Comment:
I didnt' think that the parameters starting with cache are actually caches until I looked in the code. Could you please rename them to: PkPidCache etc? Also for the ArgPeerShardMapper PeerIdPk, FallbackPkShard,FallbackPidShard I thought they are Pk -> byte[], Shard -> uint while all of them are caches. Are they actually Fallbacks?

Reply:
Ok, will rename those.
The fallback caches are mainly used for observers or for validators if the NodesCoordinator fails to provide the correct mapping between Pk and ShardID. That is why I called them fallback.

Reply:
done

Reply:
ok